Menus/MainMenu/MainMenu.cs

```diff
@@ -363,6 +374,24 @@ public void RemoveSettingsMenuItem(ISettingsMenuItemProvider provider)
             }
             provider.settingsMenuItemInstance = null;
         }
+
+        void ShowFeedback()
+        {
+            var tooltipText = proxyType == typeof(ViveProxy) ? "Press to Rotate Menu" : "Rotate Menu";
+            List<VRInputDevice.VRControl> controls;
+            if (m_Controls.TryGetValue("FlickFace", out controls))
```

Comment:
The hardcoded strings here are freaking me out. Schoen, Dylan, and I were talking about adding a scriptable object field to action source on the action map, which would contain the control name, tooltip string, and any other data we need associated with the affordance.

Reply:
It sounds like we pretty much want to make everything about Feedback serializable. This is fine for our implementation, but I want to make sure this hard-coded path still works, since there is less friction to just getting some feedback up and running this way. Same goes with the repeated code: If we are just reading serialized tooltips from actions, that will all go away.

I am a little worried about coupling proxy feedback to actionmaps, but I think because this isn't all feedback ever, it makes sense that controller feedback relies on the input system.

I think we should have one last group conversation about the feedback-in-actionmaps plan, specifically to get input from @AndrewTHEManeri before proceeding.

Reply:
Tooltip flicker fixed. Good catch 👍

Reply:
I agree with a more serialized approach to this - does it need to be this PR or can it be a subsequent one?

Reply:
I'd say definitely not. It will overlap with Dylan's PR, so we should wait until that stabilizes.